### PR TITLE
feat: add watchlater , liked videos and history videos api calls

### PR DESCRIPTION
**Pull request by Bhakti199**
here I added api calls for watch later, history videos and liked videos. 

login details: 
email: adarshbalika@gmail.com
password: adarshBalika123,

### CODE REVIEW

**Review by tushartiwari7 — APPROVED** [review]
All the Functionalities looks good to me.
Everything is working as expected.
Suggestion: Try using DRY methodology in AuthContext as many things seems repetitive.

File: src/ApiCalls/WatchLaterVideoCalls/DeleteWatchLaterVideoCall.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  try {
+    const { data, status } = await axios.delete(
+      `/api/user/watchlater/${videoId}`,
+      {
+        headers: {
+          authorization: localStorage.getItem("userLoginToken"),
+        },
+      }
+    );
+    console.log(data.watchlater, status);
```
Comment (rkapoor10) [inline src/ApiCalls/WatchLaterVideoCalls/DeleteWatchLaterVideoCall.js]: remove console.log

File: src/ApiCalls/WatchLaterVideoCalls/PostWatchLaterVideoCall.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      {
+        video,
+      },
+      {
+        headers: {
+          authorization: localStorage.getItem("userLoginToken"),
+        },
+      }
+    );
+    console.log(data, status);
```
Comment (rkapoor10) [inline src/ApiCalls/WatchLaterVideoCalls/PostWatchLaterVideoCall.js]: here also

**Review by rkapoor10 — APPROVED** [review]
- playlist title should be in white color it's in black
- add cursor pointer to the add btns in watchlater, like videos and add to playlist
- clicking on one menu icon should close others
ss

**Pull request merged**